addons/invert-stage/userscript.js

```diff
@@ -0,0 +1,37 @@
+export default async function ({ addon, console, msg }) {
+  // INVERT STAGE ADDON BY @BATMANGREEN123 (MICHEALTHERATZ)
+  const vm = addon.tab.traps.vm;
+  const canvas = vm.renderer.canvas;

```

Comment:
This part of the code errors for me, with `Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'canvas')`.

Reply:
what browser are you using?

Reply:
Edge 120 - that runs on Chromium, so if it works on Chrome, it should work here

Reply:
that's weird, maybe try it on regular chrome.

Reply:
I can't, because of parental controls - addons should work on Edge anyways.

Reply:
You have parental controls? I might have to look into this. Maybe it's just a you problem.

Reply:
because the VM object should have the canvas

Reply:
The error message seems to suggest that the VM does not have a renderer. Maybe it only exists once the project has loaded? Does this addon work for you on, say, Appel?